code/game/machinery/doors/door.dm

```diff
@@ -103,6 +103,8 @@
 	set_density(0)
 	update_nearby_tiles()
 	QDEL_NULL(dummy)
+	var/turf/T = get_turf(src)
+	T.pathweight = initial(T.pathweight)
```

Comment:
re: other comment about messing with pathweight - changing the Distance proc also means you don't have to do stuff like this. 

There's probably cases where initial() will do something invalid here (e.g. turfs loaded on runtime using spawners - initial() will set them to the _compile-time_ value, not the value the spawner created them with).